### PR TITLE
[62] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by bint-saad**
[62] Create & Style the Customer Signup Page using CSS
[62] Create & Style the Customer Signup Page using CSS
css4
css1
css2
css3

### CODE REVIEW

File: submissions/hadizaSaadu/css/signup-page.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/hadizaSaadu/css/signup-page.html:1]: KIndly place the file inside the `hmtl-forms` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @bint-saad , the padding-left and right inside the card inside the same. There is more padding on the right than on the left. Make them be even

**Comment by adaugochi** [comment]
send a new screenshot of your work

**Comment by bint-saad** [comment]
css3
css4
css1
css2

**Comment by bint-saad** [comment]
screenshot attched!!! Thanks

**Comment by adaugochi** [comment]
Well done @bint-saad , the padding-left and right inside the card inside the same. There is more padding on the right than on the left. Make them be even

**Comment by adaugochi** [comment]
image

**Comment by adaugochi** [comment]
See that left and right marking that I did. that's where I am pointing at in my comment

**Comment by bint-saad** [comment]
cssn4
cssn1
cssn2
cssn3

**Comment by bint-saad** [comment]
comment fixed...thanks

File: submissions/hadizaSaadu/html-forms/css/signup-page.css [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+  letter-spacing: normal;
+  text-align: left;
+  color: #ffffff;
+  text-decoration: none;
+}
+#sign {
+  width: 64px;
+  height: 19px;
+  margin: 12.3px 0 0;
+  font-family: "montserrat", sans-serif;
```
Comment (adaugochi) [inline submissions/hadizaSaadu/html-forms/css/signup-page.css:61]: Declare font-family in your `body` tag and remove it where it is in use elsewhere

**Review by adaugochi — CHANGES_REQUESTED** [review]
Styling like this `font-weight: normal;
  font-stretch: normal; letter-spacing: normal;
  font-style: normal;` are `normal` by default. There is no need for using them everywhere in your stylings. Kindly remove it.

Also, images should be placed in the `images` folder that is in your `html` folder.

Note that you have same files appearing multiple places. Kindly click [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/924/files) to look at what your push as PR

**Comment by bint-saad** [comment]
cssn4
cssn1
cssn2
cssn3

**Comment by bint-saad** [comment]
all existing comments fixed... kindly  review closely for any other error(s)

File: submissions/hadizaSaadu/html-forms/css/signup-page.css [hunk 3]
```diff
@@ -0,0 +1,284 @@
+/* body {
```
[the remark this answers is not in the document]
Comment (adaugochi) [reply submissions/hadizaSaadu/html-forms/css/signup-page.css:1]: You haven't fix this

**Review by adaugochi — CHANGES_REQUESTED** [review]
These images are still inside `css` folder. kindly move them 
image

image

Also, you have to remove this file
image

**Comment by bint-saad** [comment]
cssn4
cssn1
cssn2
cssn3

**Comment by bint-saad** [comment]
comments fixed

File: submissions/hadizaSaadu/html-forms/css/signup-page.css [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
```
Comment (adaugochi) [inline submissions/hadizaSaadu/html-forms/css/signup-page.css:1]: Kindly place this `signup-page.css` inside `css` folder and the `css` folder should not be inside `html-forms` folder. Check your slack message I already gave you guidelines on what you need to to

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
Why did you remove your CSS file completely? I only said you should put it in the right folder

**Comment by bint-saad** [comment]
cssn4
cssn1
cssn2
cssn3

**Comment by bint-saad** [comment]
cssn4
cssn1
cssn2
cssn3

**Comment by bint-saad** [comment]
done

**Review by adaugochi — APPROVED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**